spec/features/admin/members/admin_search_member_spec.rb

```diff
@@ -20,9 +20,9 @@
     scenario "Search with non-matching data" do
       fill_in('Email', with: current_user.email)
       click_button('Search')
-      expect(page).to have_content("No users matched that search")
+      expect(page).to have_content("No users matched your search")
```

Comment:
Maybe - but we'd have to find a nice way to display any given search results (nested or otherwise) to a user. Cost/Benefit is not there in my opinion.